### PR TITLE
Updated subscribe param to use tuple path, + tests

### PR DESCRIPTION
**Pull request by JamesYFC**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,9 +1,10 @@
+import { EventEmitter } from 'events';
```
Comment (kalvinpearce) [inline src/index.ts]: Pretty sure this is unused and might have been left over from the merge?

**Comment by JamesYFC** [comment]
Probably, will do it when I get home. Also will take a look at overloading possibilities with the path param

File: src/index.ts (hunk 1 above)
Comment (JamesYFC) [reply src/index.ts]: Probably, will do it when I get home.

**Comment by JamesYFC** [comment]
> 
> 
> Also will take a look at overloading possibilities with the path param

nvm, not needed

File: src/index.ts (hunk 1 above)
Comment (JamesYFC) [reply src/index.ts]: done

**Comment by JamesYFC** [comment]
minor cleanups

**Pull request merged**